src/ServiceControl.Audit.Persistence.Tests.RavenDB/LicenseTest.cs

```diff
@@ -37,7 +37,7 @@ public async Task EnsureLicenseIsValid()
                         Assert.That(details.Status, Is.EqualTo("Commercial"));
                         Assert.That(details.Expired, Is.False);
                         Assert.That(details.Type, Is.EqualTo("Professional"));
-                        Assert.That(DateTime.UtcNow.AddDays(14), Is.LessThan(details.Expiration), $"The RavenDB license expires {details.Expiration} which is less than 2 weeks. Contact RavenDB at <sales@ravendb.net> for the new license.");
+                        Assert.That(details.Expiration, Is.LessThan(DateTime.UtcNow.AddDays(14)), $"The RavenDB license expires {details.Expiration} which is less than 2 weeks. Contact RavenDB at <sales@ravendb.net> for the new license.");
```

Comment:
I think the way it was is correct. The idea is that the actual is two weeks from now and the expected is the expiration date to make the assertion fail when the license is about to expire. We flipped things now. If you want to flip expected and actual you need to adjust the check accordingly